desktop-app/src/renderer/components/Previewer/Guides/index.tsx

```diff
@@ -0,0 +1,194 @@
+      verticalGuidesRef?.current?.getRulerScrollPos() + coordinates.deltaY;
+
+    let scrollPosY: number;
+    if (Number(scrollY) > 0) {
+      scrollPosY =
+        addjustedInnerHeight - Number(scrollY) <= 0
+          ? addjustedInnerHeight
+          : scrollY || addjustedInnerHeight;
+    } else {
+      scrollPosY = 0;
+    }
+
+    let scrollPosX: number;
+    if (Number(scrollX) > 0) {
+      scrollPosX =
+        Number(scrollX) > coordinates.innerWidth - Number(scrollX)
+          ? coordinates.innerWidth
+          : scrollX || coordinates.innerWidth;
+    } else {
+      scrollPosX = 0;
+    }
+    if (
+      coordinates.innerHeight >= scaledHeight * zoomFactor &&
+      coordinates.innerHeight - height > scrollPosY
+    ) {
+      verticalGuidesRef.current?.scroll(scrollPosY);
+      horizontalGuidesRef.current?.scroll(scrollPosX);
+      verticalGuidesRef.current?.scrollGuides(scrollPosX);
+      horizontalGuidesRef.current?.scrollGuides(scrollPosY);
+    }
+  });
+
+  return (
+    <>
+      {enabled ? (
+        <>
+          <div
+            style={{
+              position: 'absolute',
+              top: 0,
+              left: 0,
+              pointerEvents: 'none',
+              width: `${scaledWidth + 30}px`,
+              height: `${scaledHeight + 30}px`,
+              zIndex: 1,
+              overflow: 'hidden',
+            }}
+          >
+            <div className="box bg-slate-200 dark:bg-slate-800" />
+            <Guides
+              ref={horizontalGuidesRef as LegacyRef<Guides>}
+              type="horizontal"
+              backgroundColor="transparent"
+              textColor={night ? 'rgb(209, 213, 219)' : 'rgb(55, 65, 81)'}
+              mainLineSize="40%"
+              longLineSize="5"
+              shortLineSize="1"
+              lineColor={night ? '#fefefe' : '#777777'}
+              zoom={zoomFactor}
+              style={{
+                height: '30px',
+                left: '30px',
+                width: scaledWidth * 2,
+                pointerEvents: 'auto',
+              }}
+              displayDragPos
+              displayGuidePos
+              useResizeObserver
+              defaultGuides={defaultsHor.length > 0 ? defaultsHor : undefined}
+              onChangeGuides={({ guides }) => {
+                window.electron.store.set('userPreferences.guides', [
+                  ...window.electron.store
+                    .get('userPreferences.guides')
+                    .filter((x: DefaultGuide) => {
+                      if (x.resolution !== `${width}x${height}`) {
+                        return true;
+                      }
+                      return x.is_vertical;
+                    }),
+                  {
+                    resolution: `${width}x${height}`,
+                    is_vertical: false,
+                    positions: guides.filter((x) => x > 0),
+                  },
+                ]);
+              }}
+            />
+            <Guides
```

Comment:
Any idea why the numbers are getting cut off in iPhone 12 Pro:
<img width="333" alt="Screenshot 2023-11-25 at 7 40 39 PM" src="https://github.com/responsively-org/responsively-app/assets/1283424/8cb6f70f-68ea-43f5-88d4-216a6f793d20">

Looks good on other wider screen devices though.